includes/class-sensei-grading.php

```diff
@@ -663,12 +672,12 @@ public function get_redirect_url() {
 		$course_id = intval( $lesson_data['course_id'] );
 		$grading_view = sanitize_text_field( $lesson_data['view'] );
 
-		$redirect_url = '';
 		if ( 0 < $lesson_id && 0 < $course_id ) {
-			$redirect_url = esc_url_raw( apply_filters( 'sensei_ajax_redirect_url', add_query_arg( array( 'page' => $this->page_slug, 'lesson_id' => $lesson_id, 'course_id' => $course_id, 'view' => $grading_view ), admin_url( 'admin.php' ) ) ) );
-		} // End If Statement
+			echo esc_url_raw( apply_filters( 'sensei_ajax_redirect_url', add_query_arg( array( 'page' => $this->page_slug, 'lesson_id' => $lesson_id, 'course_id' => $course_id, 'view' => $grading_view ), admin_url( 'admin.php' ) ) ) );
+		} else {
+			echo '';
```

Comment:
Is this necessary?

Reply:
I got rid of the `$redirect_url` variable, which was initialized to an empty string. So if the first condition is not met, it still needs to return an empty string.

Reply:
I'd prefer to leave it, just in case.